herbie/info.rkt

```diff
@@ -0,0 +1,11 @@
+#lang info
+(define collection "herbie")
+(define compile-omit-paths '("test" "reports/bash-pred-test.rkt" "herbie-web" "util.rkt"))
+(define raco-commands '(("herbie" (submod herbie/interface/inout main) "improve floating point expression accuracy" 75)))
+(define deps '("base"
+               "math-lib"
+               "plot-lib"
+               "profile-lib"
+               "rackunit-lib"
+               "srfi-lite-lib"
+               "web-server-lib"))
```

Comment:
I don't think we rely on the web-server except in the `demo.rkt` script and in `herbie-web`, both of which we're excluding.